### PR TITLE
1.32 release team slack groups

### PR DESCRIPTION
**Pull request by fsmunoz**
Adds the release team to Slack groups for v1.32 cycle.

xref: 
 - https://github.com/kubernetes/community/pull/7919
 - https://github.com/kubernetes/community/pull/7682

cc: @kubernetes/sig-release-leads @Priyankasaggu11929 @katcosgrove @gracenng

### CODE REVIEW

**Comment by fsmunoz** [comment]
As per the discussion in #7919  , I've removed the sig-release-leads _section_ from the file; I wasn't sure if the "remove the release leads" also meant removing the users themselves from the sections, so I erred in the side of caution @justaugustus .

File: communication/slack-config/sig-release/usergroups.yaml
Comment (Priyankasaggu11929) [inline communication/slack-config/sig-release/usergroups.yaml:70]: @fsmunoz, the change needed is following:

that is – update the  `release-team-leads` slack group to remove SIG Release Leads (chair/TLs), and add Grace/Kat (as SIG Release sub-project leads)

```suggestion
      - chanieljdan         # 1.32 Docs Lead
      - drewhagen           # 1.32 Release Signal Lead
      - fsmunoz             # 1.32 Release Lead
      - gracenng            # Release Team Subproject Lead
      - katcosgrove         # 1.32 EA + Release Team Subproject Lead
      - mbianchidev         # 1.32 Communications Lead
      - npolshakova         # 1.32 RT Lead Shadow
      - salehsedghpour      # 1.32 RT Lead Shadow
      - satyampsoni         # 1.32 Release Notes Lead
      - sreeram-venkitesh   # 1.32 RT Lead Shadow
      - tjons               # 1.32 Enhancements Lead
      - Vyom-Yadav          # 1.32 RT Lead Shadow
```

File: communication/slack-config/sig-release/usergroups.yaml
Comment (Priyankasaggu11929) [inline communication/slack-config/sig-release/usergroups.yaml:68]: @fsmunoz, this section (`sig-release-leads` user group) needs to be re-added.

**Review by Priyankasaggu11929 — CHANGES_REQUESTED** [review]
@fsmunoz, left a few inline comments for requesting changes. Thanks!

File: communication/slack-config/sig-release/usergroups.yaml
Comment (fsmunoz) [reply communication/slack-config/sig-release/usergroups.yaml:70]: Thank you, so it's sort of the opposite option that I knew existed :D

File: communication/slack-config/sig-release/usergroups.yaml
Comment (fsmunoz) [reply communication/slack-config/sig-release/usergroups.yaml]: Done.

**Comment by Priyankasaggu11929** [comment]
/lgtm
/approve

/hold
for any further reviews

Thanks @fsmunoz!

**Comment by Priyankasaggu11929** [comment]
unholding to merge, since it's blocking the current release team leads to receive the intended slack notifications.

/unhold.

**Pull request merged**
